optimizely/optimizely.py

```diff
@@ -101,6 +101,11 @@ def __init__(self,
     # TODO: remove it and delete test_event_builder file
     self.event_builder = event_builder.EventBuilder()
     self.decision_service = decision_service.DecisionService(self.logger, user_profile_service)
+    self._disposed = False
+
+  @property
```

Comment:
why `@property` is needed, jus want to understand.

Reply:
To make it readonly that can only be updated from inside the class but not from outside the class.